site/.vitepress/config.ts

```diff
@@ -121,6 +121,10 @@ export default defineConfig({
             text: "How to Sponsor Gas for a User Operation",
             link: "/sponsoring-gas",
           },
+          {
+            text: "How to Check if a User Operation is Eligible for Sponsorship",
```

Comment:
This is likely going to spill over in the sidebar

Reply:
wonder if it makes sense to have this be a header within the How To Sponsor Gas section instead

Reply:
sidebar wraps long lines, it already does for all guides sections where titles are all long